app/post.go

```diff
@@ -1750,6 +1750,121 @@ func (a *App) GetTopThreadsForUserSince(teamID, userID string, opts *model.Insig
 	return topThreadsWithEmbedAndImage, nil
 }
 
+func (a *App) SetPostReminder(postID, userID string, targetTime int64) *model.AppError {
+	// Store the reminder in the DB
+	reminder := &model.PostReminder{
+		PostId:     postID,
+		UserId:     userID,
+		TargetTime: targetTime,
+	}
+	err := a.Srv().Store.Post().SetPostReminder(reminder)
```

Comment:
Would it be a good practice to check whether a post does exist or not before saving the reminder?

Reply:
Yeah good point. We can use the `RowsAffected` field for this. I can make the changes.